<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPage</class>
 <widget class="QWidget" name="AdminPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 24px;
	font-weight: bold;
	
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Homepage</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="info_mail">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 10px;
	font-weight: bold;
	
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Mail :</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_login">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 10px;
	font-weight: bold;
	
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Login :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_deconnect">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>450</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius: 20px;
	
	background-color: rgb(255, 15, 19);
	font-weight:  bold;
	font-size: 15px;
	color: rgb(255, 255, 255);
}

QPushButton:hover{
	
	background-color: rgb(185, 1, 5);
}</string>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createUserButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>450</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius: 20px;
	
	background-color: rgb(255, 15, 19);
	font-weight:  bold;
	font-size: 15px;
	color: rgb(255, 255, 255);
}

QPushButton:hover{
	
	background-color: rgb(185, 1, 5);
}</string>
   </property>
   <property name="text">
    <string>Create User</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_firstName">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 10px;
	font-weight: bold;
	
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>First Name : </string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="info_lastName">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 10px;
	font-weight: bold;
	
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Last Name :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
